docs/standard-library/vectorized-stl-algorithms.md

```diff
@@ -30,6 +30,8 @@ The manually vectorized algorithms use template meta-programming to detect the s
 
 Generally, programs either benefit in performance from this manual vectorization or are unaffected by it. In case of any problem, you can disable manual vectorization by defining the `_USE_STD_VECTOR_ALGORITHMS` macro set to 0. It defaults to 1 on x64 and x86, which means that manually vectorized algorithms are enabled by default.
 
+When overriding `_USE_STD_VECTOR_ALGORITHMS` make sure to set the same value for all linked translation units that use algorithms. Reliable way to achieve that is using project properties rather than defining it in the source.
```

Comment:
Could you provide the steps for how to set this in the project. Since that's the main direction we want them to go, we should show them how. It also helps reassure the reader that they are doing it right if they aren't totally clear on what it means to set project properties.